bootstrap/scripts/2-download.sh

```diff
@@ -23,12 +23,13 @@ function download_to {
 }
 
 if [ ! -e "${BOOTSTRAP_VMTARGET}" ]; then
-    # Downloads a SPUR vm for the configured architecture
-    mkdir ${BOOTSTRAP_DOWNLOADS}/vmtarget
-    cd ${BOOTSTRAP_DOWNLOADS}/vmtarget
-    ${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 90 vm $BOOTSTRAP_ARCH
-    cd -
-    echo "Target VM: $(${VM} --version)"
+	rm -rf "${BOOTSTRAP_DOWNLOADS}/vmtarget"
+	# Downloads a SPUR vm for the configured architecture
+	mkdir ${BOOTSTRAP_DOWNLOADS}/vmtarget
+	cd ${BOOTSTRAP_DOWNLOADS}/vmtarget
+	${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 110 vm $BOOTSTRAP_ARCH
```

Comment:
I think the entire PR is good.
The only line that makes me a bit worried is this one indeed.
I'd like to build (unstable) version X with a stable version <X.

So I'd feel better if we put Pharo 10 in here. What do you think?

Reply:
If a new primitive was added to the VM and a new method using that primitive was added to the image, then they should be used together. Trying to debug an image with a new primitive without VM support would be difficult. It seems to me that they should all stay together.

Reply:
Yes - but at the moment we do not have this as a case, right? So maybe it makes sense to go with the last *stable* until we have to solve such a situation and have a need to go the *unstable* route. No?

Reply:
I guess I don't feel that strongly, but it seems that we also do not have the case where the 11 VM doesn't work. So maybe go with the latest VM "until we have to solve such a situation." Also, it seems that if we need the latest VM and it doesn't come in automatically, it will not be very obvious where to go look to solve the problem. So, which is more likely, the latest VM is broken or the latest VM is needed but missing? Also, my impression is that the VM is on version 9 and it is being used by several versions of Pharo. If the same thing is being used for Pharo 9, 10, and 11, what advantage is there in using the Pharo10 VM if it is the same as 9 and 11? If they are going to use the same thing (which I don't know), then the numbers should match. But, again, my interest is in other parts of the script; I don't care that much here and can change if it will help get the rest accepted.

Reply:
Discussion is mainly about the risk through dependencies: if we have the stable VM for the bootstrap we would go with less risk. When using stable with < 10 for the bootstrap the VM is not the moving target and the bootstrap process should work on that end. 
That means contributions can be done for image and bootstrap Smalltalk code more savely (usually we have more contributions to image side than to VM code). 

Only when situation would be needed to change all three at the same time: VM + image + boostrap for a new feature introduction (primitive, ....) the unstable version (here X in our case now) would make sense. At least that is how I see it.

Reply:
Changed to VM 100 as requested.